performance/performance_tuning.rst

```diff
@@ -152,6 +152,14 @@ Differences in worker execution can be caused by tuning configuration difference
 
   Note that when citus.explain_all_tasks is enabled, EXPLAIN plans are retrieved sequentially, which may take a long time for EXPLAIN ANALYZE.
 
+For troubleshooting and detecting the bottlenecks, you can use this configuration variable:
+
+.. code-block:: postgresql
+
+  SET citus.explain_analyze_sort_method = 'execution-time';
```

Comment:
The reference section (and Citus source) says that execution-time is the default value, so do we need to tell people to set the GUC this way? Perhaps we simply mention that the results of explain analyze will be sorted by the length of their execution time, and that the sort order is configurable (linking to the reference section).